scaletest/templates/scaletest-runner/main.tf

```diff
@@ -0,0 +1,434 @@
+terraform {
+  required_providers {
+    coder = {
+      source  = "coder/coder"
+      version = "~> 0.11"
```

Comment:
I suppose we could change it to `~> 0.11.2` so that we only allow patch versions. I’m currently including a separate terraform lock file though. Or do you think it’s better to lock it to a single version here?

Reply:
We'd otherwise need to keep this in sync with the latest version of terraform-provider-coder. My main concern here is the possibility missing some issue because we were using an old provider version, although I'm struggling to imagine a scenario where that could occur. Your call, and we can probably bump to a follow-up PR.

Reply:
Ah, I think I misunderstood. I thought you were suggesting we lock this down further since `~> 0.11` already allows minor version upgrades, e.g. 0.12 etc. I think a compromise may be to keep this lax and instead update the separate lock file each time this template is automatically updated (via push to `main`?). That way we won't suddenly break scale tests just because someone pushed a change to the provider in the middle of a development cycle.